doc/source/dev/howto-docs.rst

```diff
@@ -215,6 +215,218 @@ Note that for documentation within NumPy, it is not necessary to do
 Please use the ``numpydoc`` :ref:`formatting standard <numpydoc:format>` as
 shown in their :ref:`example <numpydoc:example>`.
 
+.. _doc_c_code:
+
+Documenting C/C++ Code
+======================
+
+NumPy uses Doxygen_ to parse specially-formatted C/C++ comment blocks. This generates
+XML files, which are  converted by Breathe_ into RST, which is used by Sphinx.
+
+**It takes three steps to complete the documentation process**:
+
+1. Writing the comment blocks
+-----------------------------
+
+Although there is still no commenting style set to follow, the Javadoc
+is more preferable than the others due to the similarities with the current
+existing non-indexed comment blocks.
+
+.. note::
+   Please see `"Documenting the code" <https://www.doxygen.nl/manual/docblocks.html>`__.
+
+This is how Javadoc style looks like::
```

Comment:
This is _what_ the Javadoc style  looks like

Reply:
done, thanks.